docs/notebooks/dispersion/two_fluid_dispersion_solver.ipynb

```diff
@@ -0,0 +1,271 @@
+{
```

Comment:
<p>Replace the cell with...</p><p> </p><p><code></code><code></p><p>%matplotlib inline</p><p> </p><p>import astropy.units as u</p><p>import matplotlib.pyplot as plt</p><p>import numpy as np</p><p> </p><p>from matplotlib import colors</p><p>from mpl_toolkits.axes_grid1 import make_axes_locatable</p><p> </p><p>from plasmapy.formulary import parameters as pfp</p><p>from plasmapy.dispersion.two_fluid_dispersion_solver import two_fluid_dispersion_solution</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/932/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>